src/app/shared/selectors/entry-file-selector.ts

```diff
@@ -154,4 +162,40 @@ export abstract class EntryFileSelector implements OnDestroy {
       });
     }
   }
+
+  /**
+   * Retrieve the correct validation messages
+   * @param isDescriptor Yes if looking at descriptor, false otherwise (test params)
+   * @param version The version of interest
+   */
+  checkIfValid(isDescriptor: boolean, version: any): void {
+    let fileEnum = null;
+    if (this.currentDescriptor === ToolDescriptor.TypeEnum.CWL) {
```

Comment:
How does this break with Nextflow (or future languages?)
(Maybe it doesn't and just returns actually)